net/acme-common/files/acme.init

```diff
@@ -39,8 +39,17 @@ load_options() {
 	export domains
 	export main_domain
 	main_domain="$(first_arg $domains)"
-	config_get keylength "$section" keylength ec-256
-	export keylength
+	config_get keylength "$section" keylength
+	if [ "$keylength" ]; then
+		log warn "Option \"keylength\" is deprecated, please use key_type (e.g., ec256, rsa2048) instead."
+		case $keylength in
+		ec-*) key_type=${keylength/-/} ;;
+		*) key_type=rsa$keylength ;;
+		esac
+	else
+		config_get key_type "$section" key_type ec256
```

Comment:
Hmm, if someone actually sets the new value, the old one probably shouldn't override it, should it?

Reply:
I assumed the convention was that if the old value exists, a user probably hasn’t completely migrated yet.

If you want to change that to “as soon so the new value comes into being“ I’m fine with that too.

Let me know.

Reply:
Hmm, I guess we also do the "old name first, then new" for the `staging` option, so let's just keep it here as well to stay consistent...